Main/Algoritm/T3(column)/T3(column).cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include <iomanip>
+#include "column_major_order.h"
+
+#define MIN_SIZE uint_fast8_t
```

Comment:
better use `using`, it's safer alternative to #define
and why is it called MIN_SIZE? 😄 
I would go with `uint8_t` or `byte_t`